src/source/Stun/Stun.h

```diff
@@ -21,7 +23,6 @@ extern "C" {
 #define STUN_HEADER_LEN                (UINT16) 20
 #define STUN_HEADER_TYPE_LEN           (UINT16) 2
 #define STUN_HEADER_DATA_LEN           (UINT16) 2
-#define STUN_HEADER_MAGIC_COOKIE       (UINT32) 0x2112A442
```

Comment:
If this is defined in the stun dependency, why do we have the `STUN_HEADER_MAGIC_COOKIE_LEN` defined here instead of the STUN library?